usaspending_api/api_docs/api_documentation/financial_spending/major_object_class.md

```diff
@@ -0,0 +1,66 @@
+
+## Financial Spending Major Object Class
+**Route**: `/api/v2/financial_spending/major_object_class?fiscal_year=[year]&funding_agency_id=[id]`
+
+Retrieve Award Spending Amounts for all Recipients with Respective Top Tier and Sub Tier Agencies
+
+**Method**: `GET`
+
+This route sends a request to the backend to retrieve awarded amounts for all recipients for a specified awarding agency based on the fiscal year, and optional award category.
+
+`Awarding Agency may be a Sub-Tier Agency.`
```

Comment:
Not relevant, should remove

Reply:
removed `Awarding Agency may be a Sub-Tier Agency.`